p1_mqtt/mqtt.py

```diff
@@ -92,6 +92,9 @@ def mqtt_on_disconnect(client: mqtt.Client, userdata: Any, rc: int) -> None:
 
         break
 
+    rate = config["mqtt_rate"]
+    last_message = time.monotonic()
```

Comment:
```suggestion
    last_message = 0
```
This should make it more likely that the first telegram received will be sent instead of waitng for one `config["mqtt_rate"]` period.